doc/api/os.md

```diff
@@ -202,6 +202,8 @@ added: v2.3.0
 
 The `os.homedir()` method returns the home directory of the current user as a
 string.
+Calling this might rarely result in an exception due to e.g. `ENOENT` and other
```

Comment:
Do you think it makes sense to document something like that in a more generic way? I expect quite a few people not to think about that when they can't find anything like that in the documentation.